.env.sample

```diff
@@ -43,4 +44,5 @@ STATSD_BACKEND=markus.backends.logging.LoggingMetrics
 STATSD_HOST=
 STATSD_PORT=
 STATSD_PREFIX=experimenter
-USE_GOOGLE_ANALYTICS=false
\ No newline at end of file
+USE_GOOGLE_ANALYTICS=false
+SKIP_REVIEW_ACCESS_CONTROL_FOR_DEV_USER=true
```

Comment:
This setting has been really handy for local dev: repeatedly changing the `changedBy` in admin to unlock review access started getting old after the first few rounds.

Reply:
This is SO dang handy